classes/netobj.adoc

```diff
@@ -12,6 +12,9 @@ If name of the custom attribute overlaps with the instance attribute, method <<c
 `alarms => array`::
 List of active <<class-alarm, Alarms>> for this object.
 
+`alarmsStats => array`::
+Statistics of active <<class-alarm, Alarms>> for this object.
```

Comment:
It does not return Alarm class objects. IT returns Integer array that consists of 5 numbers - each number represents number of alarms for this and child objects for each of severity.

Reply:
and mistake in naming alarmsStats -> should be alarmStats